coco/master.py

```diff
@@ -52,15 +54,18 @@
 
     with await redis.conn as r:
 
+        # Check if queue is full
+        if QUEUE_LEN > 0 and await r.llen("queue") >= QUEUE_LEN:
+            # Increment dropped request counter
+            await r.incr(f"dropped_counter_{endpoint}")
+
+            return response.json({"reply": "Coco queue is full.", "status": 503}, status=503)
```

Comment:
I debated whether 429 was a better error here, the meaning "Too many requests" is a bit more explicitly what we want, but I think it's really meant to be when a *single* client sends too many requests and gets throttled. So 503 seems good.